vendor/modules.txt

```diff
@@ -8,13 +8,15 @@ github.com/asaskevich/govalidator
 github.com/dave/jennifer/jen
 # github.com/davecgh/go-spew v1.1.1
 github.com/davecgh/go-spew/spew
+# github.com/fatih/color v1.9.0
```

Comment:
Honestly, I think I would be happy removing vendoring from this project altogether. The module ecosystem is solid enough now that I don't think we need it.